### DISCUSSION TITLE
Can we "archive" bills so they no longer appear in the "Bills To Pay" dashboard?

### DISCUSSION THREAD

**Discussion by oliverseal** [firefly-iii/firefly-iii#4438] [Firefly III]
**Description**

Currently when viewing the dashboard (which is very useful), the "Bills To Pay" section includes any bills that are "expected". This is cool, but if a service has been canceled or a bill is "paid off", then this number will be misleading.

I'd love it if we could mark a bill as "Archived" and it doesn't show up in this calculation any more.

**Solution**
- Add a boolean field to bills, `is_archived` (or `is_active` to follow the style used on accounts), and that excludes it from further consumption.
- The rules are rules, so this would have no change on them, I think.

**What are alternatives?**
Mental math to know that it's 82.61 I owe, not 98.84, etc.

If there's already a way to do this, then yay! Please inform me.

If not, I may put together a PR if I mine up some free time from the depths.

**Comment by JC5** [comment]
There's an active checkbox when you edit the bill.

**Reply by oliverseal** [reply]
My god. I feel like such an idiot. It's right there. It's looking me right in my stupid face.
Sorry to waste your time.

**Reply by JC5** [reply]
No worries, it is kind of hidden :D
